controllers/pwa.js

```diff
@@ -118,6 +114,7 @@ router.post('/add', (req, res, next) => {
       return pwaLib.createOrUpdatePwa(pwa);
     })
     .then(savedData => {
+      cache.flush();
```

Comment:
we should flush only the lists, I need to find if I can get the cache.keys to look for them or store them somewhere when they get cached.